diff --git a/node_modules/react-native-reanimated/src/reanimated1/core/AnimatedNode.js b/node_modules/react-native-reanimated/src/reanimated1/core/AnimatedNode.js
index d7892d8..5d19c3e 100644
--- a/node_modules/react-native-reanimated/src/reanimated1/core/AnimatedNode.js
+++ b/node_modules/react-native-reanimated/src/reanimated1/core/AnimatedNode.js
@@ -77,7 +77,7 @@ function runPropUpdates() {
 }
 
 const scheduleUpdates =
-  Platform.OS === 'web' ? requestAnimationFrame : setImmediate;
+  Platform.OS === 'web' ? requestAnimationFrame : a => a();
 
 export default class AnimatedNode {
   __nodeID;
